backends/qualcomm/runtime/QnnExecuTorchBackend.cpp

```diff
@@ -60,124 +59,19 @@ Result<DelegateHandle*> QnnExecuTorchBackend::init(
       "Fail to initialize Qnn Manager");
 
   if (qnn_manager->IsOnlinePrepare()) {
-    auto graph = qcir::GetGraph(qnn_context_blob.buffer);
```

Comment:
where does the online prepare option go?

Reply:
If I understand the question correctly, the `online_prepare` is still working but the processing logic is refactored into `CompileQcir` which could also be leveraged in compiling `multiple_graphs`.